cli/inputhandler/options.cpp

```diff
@@ -0,0 +1,64 @@
+#include "options.h"
+
+CLIOptions::CLIOptions() :
+        m_optionsTx("transaction", "Manage transactions"),
+        m_optionsPem("pem", "Manage pem files")
+{
+    initOptions();
+}
+
+cxxopts::Options CLIOptions::transaction() const
+{
+    return m_optionsTx;
+}
+
+cxxopts::Options CLIOptions::pem() const
+{
+    return m_optionsPem;
+}
+
+std::string CLIOptions::help() const
+{
+    return helpTx() + "\n" + helpPem();
+}
+
+std::string CLIOptions::helpPem() const
+{
+    return m_optionsPem.help();
+}
+
+std::string CLIOptions::helpTx() const
+{
+    return m_optionsTx.help();
+}
+
+void CLIOptions::initOptions()
+{
+    initOptionsTx();
+    initOptionsPem();
+}
+
+void CLIOptions::initOptionsTx()
+{
+    m_optionsTx.add_options("new") // transaction new group
```

Comment:
Not sure, are there use cases for the CLI, in general, or is it for demonstrating the SDK usage?

Reply:
right now I think it is just for demonstrating the SDK.

Reply:
Yes, just for some functionality demonstration